src/modules/thirdParty/sagas.ts

```diff
@@ -212,6 +239,29 @@ export function* thirdPartySaga(builder: BuilderAPI) {
 
   function* handlePublishAndPushChangesThirdPartyItemRequest(action: PublishAndPushChangesThirdPartyItemsRequestAction) {
     const { thirdParty, itemsToPublish, itemsWithChanges } = action.payload
-    yield all([put(publishThirdPartyItemsRequest(thirdParty, itemsToPublish)), put(pushChangesThirdPartyItemsRequest(itemsWithChanges))])
+    const collectionId = getCollectionId(itemsToPublish)
+    // We need to execute this two in secuence, because the push changes will create a new curation if there was one approved.
+    // It will create it with status PENDING, so the publish will fail if it's executed after that event.
```

Comment:
If we're talking about the curations:
It will create `them` with status PENDING ...